### ISSUE TITLE
Lost my hours in contest!

### ISSUE THREAD

**Issue by martys00** [develsoftware/GMinerRelease#320]
Hello, I've been mining for 24 hours started yesterday, my internet connection crashed for like a minute or two minutes and I lost all my 24 hours and the contest hours reseted to zero and I must start from the beginning. Is there something to get it back or am I screwed? xD Thanks for helping me in advance, have a nice day y'all!

**Comment by s4brown** [comment]
Happens to me too anytime the miner is restarted. Running on Windows 11

**Comment by martys00** [comment]
> Happens to me too anytime the miner is restarted. Running on Windows 11

Sorry to hear that, but I'm glad I'm not the only one. Please creators of gminer, do something with it, thank you :)

**Comment by gwongis88** [comment]
I have a feeling you don't actually lost your accumulated time.  Read the 8 and 9 item point.  But would be nice it show the different session total accumulation if that is the case.   With that said, I am afraid to upgrade to 2.62 version as it said need a reboot.  LOL

**Comment by martys00** [comment]
Yes, I read the 8 and 9 point and I don't know why, but it happened as I told. I thought that the server would know my hours, but that didn't happen, when I turn off the miner and start it again, I have the hours I had earlier, but that time when my internet crashed, my hours just dissapeared and I had zero hours. I would not make an issue for something I made up, I have no need to.

**Comment by s4brown** [comment]
I have two rigs running. My main rig is for mining only and keeps accumulated hours between miner restarts or reboots. I also mine on my daily-driver and it resets each time I stop the miner. Main rig is Windows 7, daily use is Windows 11.

**Comment by martys00** [comment]
> I have two rigs running. My main rig is for mining only and keeps accumulated hours between miner restarts or reboots. I also mine on my daily-driver and it resets each time I stop the miner. Main rig is Windows 7, daily use is Windows 11.

Thank you for keeping us updated with your situation, as I said, this issue is real and I would be glad to hear any response from the creators of this contest. For the record, I have Windows 10 Pro 64bit (20H2) running.

**Comment by martys00** [comment]
> I have a feeling you don't actually lost your accumulated time. Read the 8 and 9 item point. But would be nice it show the different session total accumulation if that is the case. With that said, I am afraid to upgrade to 2.62 version as it said need a reboot. LOL

My bad I turned off the miner, where I had the 24 hours mined and the internet crash logged, I hoped that when I stop it and turn it back on I would have the hours again, but I didn't. Maybe if there are any loggs recorded from gminer I can for sure provide them here, but I don't know if gminer stores loggs. That was my big mistake I didn't screenshot it.

**Comment by zulu1905** [comment]
21 hours erased and started again from zero hour. if it's going to be reset all the time like this, I think this contest thing is fucked up.

**Comment by martys00** [comment]
> 21 hours erased and started again from zero hour. if it's going to be reset all the time like this, I think this contest thing is fucked up.

Thank you very much for posting your issue here, that is another person with the same problem, let’s make this thing big, so the creators of this contest will see it. But maybe this contest really is somehow f**ked up, because of all us somehow losing hours, let’s see if someone gonna do something about it now.

**Comment by gwongis88** [comment]
I use WIN10.  I have no problem.  Just one shot  got 72/72.  and stay as 72/72

**Comment by martys00** [comment]
> I use WIN10. I have no problem. Just one shot got 72/72. and stay as 72/72

Yes, I trust you, that everything in your system works. But some people have this problem, as you can see in this page. And I know nobody would give me back my hours, because I don't have the screenshot of it. And my goal is to share it here, so the creators of this contest see it to check it out, where is the problem.

**Comment by martys00** [comment]
Ok guys, it happened again, now even more weirdly, because I had all hours in the contest, means 72/72 and now I checked gminer and have 31/72. I hoped I would never need to deal with this problem again, but now it seems very strange and I think the contest is just bulls**t, sorry creators, but this is just obvious there is not something right. You should have my hours stored in your server, but I think your server doesn't even work properly. How you can do a contest, when you don't even have it tested on 100%. And again I cannot provide screenshot, because it happened so strangely, I woke up this morning and checked gminer and I have log where I have only 31 hours in contest. Before I had 72/72. You should check your server!

**Comment by martys00** [comment]
Very weird something like this happened right when it is 7 days from the starting of this contest. I really don't trust this.

**Comment by AlexLMX** [comment]
Same to me - it was 48/72 and then all lost 0/72 (even not restart PC)

**Comment by martys00** [comment]
> Same to me - it was 48/72 and then all lost 0/72 (even not restart PC)

Thank you for posting your issue here, I'm sorry to hear that you lost your hours. It happened to me like 3 times now, but I managed to mine all days to get 72/72 and for now it stays like that, so I must say my issue has been fixed, but I'm not saying, it could drop any time, so I will let you updated. Waiting for the results of this contest too. Please y'all have a great, happy day!

**Comment by martys00** [comment]
So, what's up with the contest guys, does anybody know? I would like to see who's the winner. Tell me.
